packages/console/src/pages/TenantSettings/TenantDomainSettings/index.tsx

```diff
@@ -16,8 +17,7 @@ function TenantDomainSettings() {
   const api = useApi();
   const fetcher = useSwrFetcher<Domain[]>(api);
   const { data, error, mutate } = useSWR<Domain[], RequestError>('api/domains', fetcher, {
-    // Note: check the custom domain status every 10 seconds.
-    refreshInterval: 10_000,
+    refreshInterval: customDomainSyncInterval * 1000,
```

Comment:
no need to customize?

Reply:
Per offline disccusion, this refreshInterval is needed, otherwise, the data won't update unless you focus or reload the page.